programming/webstorm/pspec.xml

```diff
@@ -10,7 +10,7 @@
         <PartOf>programming</PartOf>
         <Summary xml:lang="en">WebStorm - an IDE for the Web</Summary>
         <Description xml:lang="en">WebStorm - an IDE for the Web</Description>
-        <Archive type="targz" sha1sum="531881bc5760dfaf05e4d8888f89a4a9c3e4f1a9">https://download.jetbrains.com/webstorm/WebStorm-2018.1.3.tar.gz</Archive>
+        <Archive type="targz" sha1sum="22c48fcaecedb0c0f308c24645c8247275b8bf4bb60ac3b3404f092ac92c5d11">https://download.jetbrains.com/webstorm/WebStorm-2018.1.4.tar.gz</Archive>
```

Comment:
That's the sha256sum, not the sha1sum. Does this actually work on your end?

Reply:
Just for the sake of completeness, the sha1sum would be fefec3422ce2b4a3a67bf1cec86deb6e650ced9d